dali/pipeline/workspace/iteration_data.h

```diff
@@ -35,8 +37,8 @@ namespace dali {
  *   - "number_of_unprocessed_samples" -> "100"
  *   - "next_batch_ready" -> "true"
  */
-using operator_trace_map_t = std::unordered_map<
-  std::string /* trace_name */, std::string /* trace_value */>;
+using operator_trace_map_t = std::map<
+  std::string /* trace_name */, std::string /* trace_value */, std::less<>>;
```

Comment:
No templated lookup in unordered_map until C++20.

Reply:
do you  need to specify std::less ?

Reply:
yes, I need to specify `std::less<>` as opposed to the default `std::less<key_type, value_type>` - this allows the search term to be of different (but still comparable) type. In this case, you can pass `string_view` or `const char *` to `.find` without creating an intermediate `string`.